lib/apiservers/portlayer/restapi/handlers/interaction_handlers.go

```diff
@@ -91,8 +91,7 @@ func (i *InteractionHandlersImpl) ContainerResizeHandler(params interaction.Cont
 
 	err = connContainer.Resize(cWidth, cHeight, 0, 0)
 	if err != nil {
-		retErr := &models.Error{Message: "SSH Session failed to resize the container's TTY"}
-		return interaction.NewContainerResizeInternalServerError().WithPayload(retErr)
+		return interaction.NewContainerResizeInternalServerError()
```

Comment:
Ah, I'll add a log line.

This is the crux of the panic fix, btw.  The `WithPayload` calls when there is no client to receive the payload will panic the server.